packages/interfaces/src/filter.ts

```diff
@@ -34,7 +35,7 @@ export type IFilterSDK = IReceiver &
   IBaseProtocolSDK & { protocol: IBaseProtocolCore } & {
     createSubscription(
       pubsubTopicShardInfo?: ShardingParams | PubsubTopic,
-      options?: SubscribeOptions
+      protocolUseOptions?: ProtocolUseOptions
```

Comment:
nit: here and in other places - let's use `options` instead of `protocolUseOptions`

Reply:
I am not sure I follow.

I suggest:
```suggestion
      options?: ProtocolUseOptions
```